lib/services/resource.go

```diff
@@ -295,6 +297,9 @@ func (m *Metadata) Check() error {
 	if m.Name == "" {
 		return trace.BadParameter("missing parameter Name")
 	}
+	if m.Namespace == "" {
+		m.Namespace = defaults.Namespace
+	}
```

Comment:
If this function is setting values, what do you think about calling it `CheckAndSetDefaults`?